moveit_ros/moveit_servo/test/launch/test_basic_servo.test.py

```diff
@@ -0,0 +1,26 @@
+import os
+import sys
+import unittest
+import time
+import launch_testing.asserts
+
+sys.path.append(os.path.dirname(__file__))
+from servo_launch_test_common import generate_servo_test_description
+
+
+def generate_test_description():
+    return generate_servo_test_description(
+        gtest_name="basic_servo_tests",
+    )
+
+
+class TestGTestProcessActive(unittest.TestCase):
```

Comment:
Nitpick: unless this is a standard name, I think the naming is a little odd. Maybe something like:

`GTestWaitForCompletion`?

Please add a comment about what it does, too.

Also, can you move it into `servo_launch_test_common.py`? Since this same class appears several places.

Reply:
Sure, I've put a comment, changed the name and moved it to `servo_launch_test_common.py`

Reply:
Moving these classes to servo_launch_test_common seemed to break. I reverted back into files.